doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -138,6 +138,9 @@ One can define new types with the following commands.
    :n:`with @tac2typ_def`
      Permits definition of mutually recursive type definitions.
 
+   In :n:`@tac2alg_constructor`, :n:`attribute` supports :attr:`deprecated` (without `use`)
+   and :attr:`warn`.
```

Comment:
According to https://coq.inria.fr/doc/master/refman/using/libraries/writing.html , `warn` is a different attribute than `deprecated` so I just wouldn't mention it here.